docs/proposals/vertical-pod-autoscaler.md

```diff
@@ -0,0 +1,75 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/proposals/vertical-pod-autoscaler.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+## Abstract
+
+Vertical Pod Autoscaler is a feature that periodically adjust pods' [Resources](resource-qos.md#resource-specifications) based on demand signals.
+This is a natural extension of [Initial Resources](initial-resources.md) plugin.
+
+**Note:** since there is no plan to implement the feature immediately (as for 9/9/15) this document is just an overview how it should be done
+rather than detailed design.
+
+## Motivation
```

Comment:
Yes, my point is that someone reading this doc may be familiar with resource estimation (since it is in the Borg paper and also now supported in Mesos) but not with vertical autoscaling. You should clarify that this proposal is about relatively infrequent adjustments based on long history, not about resource estimation which is relatively frequent adjustments based on recent history. (To me, these are the main differences between autoscaling and resource estimation -- not whether it's tier 1 or tier 2.)